### PR TITLE
Adding parameters to the Developer pages

### PR DESCRIPTION
**Pull request by ofercloudify**

### CODE REVIEW

File: content/developer/writing_plugins/creating-your-own-plugin.md [hunk 1]
```diff
@@ -615,7 +615,7 @@ Several attributes under `ctx.plugin` can be used to access details about the pl
 
 ## Testing Your Plugin
 
-In most cases, the recommendation is to test your plugin's logic using local workflows, and only then run them as part of a Cloudify deployment. We have supplied you with a nice and tidy
+In most cases, the recommendation is to test your plugin's logic using local workflows, and only then run them as part of a {{< param product_name >}} deployment. We have supplied you with a nice and tidy
 decorator to do just that. The `cloudify-common`'s `test_utils` package enables you to do that. It is intuitive to use, and an example is provided below:
```
Comment (qooban) [inline content/developer/writing_plugins/creating-your-own-plugin.md:619]: `cloudify-common` - not sure if we can change that.

File: content/developer/writing_widgets/widgets-components.md [hunk 2]
```diff
@@ -9,60 +9,60 @@ aliases: ["/apis/widgets-components/", "/developer/custom_console/widgets-compon
 ui_components_link: "https://docs.cloudify.co/ui-components/2.1.0"
 ---
 
-You can find here documentation for all [ReactJS](https://reactjs.org/) components developed by Cloudify team. 
+You can find here documentation for all [ReactJS](https://reactjs.org/) components developed by the  {{< param product_name >}} team.
 All of those components can be accessed in custom widget development using `Stage.Basic` global object. They are divided in this page into 3 sections depending on the source:
 
-1. **[Cloudify UI Components]({{< field "ui_components_link" >}})**  - Shared React components developed by Cloudify team (**[GitHub repository](https://github.com/cloudify-cosmo/cloudify-ui-components)**)
+1. **[Cloudify UI Components]({{< field "ui_components_link" >}})**  - Shared React components developed by the {{< param product_name >}} team (**[GitHub repository](https://github.com/cloudify-cosmo/cloudify-ui-components)**)
```
Comment (qooban) [inline content/developer/writing_widgets/widgets-components.md:15]:
```suggestion
1. **[{{< param product_name >}} UI Components]({{< field "ui_components_link" >}})**  - Shared React components developed by the {{< param product_name >}} team (**[GitHub repository](https://github.com/cloudify-cosmo/cloudify-ui-components)**)
```

Should we also white-label UI Components site?

File: content/install_maintain/manager_architecture/_index.md [hunk 3]
```diff
@@ -6,43 +6,43 @@ alwaysopen: false
...
-Note that Cloudify can run in "agentless" mode, which means that agents can use specific plugins to manage hosts without the agents being installed on those hosts. You can specify which server nodes will have agents installed on them in the blueprint.
+Note that {{< param product_name >}} can run in "agentless" mode, which means that agents can use specific plugins to manage hosts without the agents being installed on those hosts. You can specify which server nodes will have agents installed on them in the blueprint.
 {{% /note %}}
 
 For more information about agents, [click here]({{< relref "install_maintain/agents/_index.md" >}}).
 
 ## {{< param cfy_console_name >}}
 
-Cloudify includes a {{< param cfy_console_name >}} that provides the same features as the CLI, as well as others. For more information, [click here]({{< relref "working_with/console/_index.md" >}}).
+Cl{{< param product_name >}}oudify includes a {{< param cfy_console_name >}} that provides the same features as the CLI, as well as others. For more information, [click here]({{< relref "working_with/console/_index.md" >}}).
```
Comment (qooban) [inline content/install_maintain/manager_architecture/_index.md:48]:
```suggestion
{{< param product_name >}} includes a {{< param cfy_console_name >}} that provides the same features as the CLI, as well as others. For more information, [click here]({{< relref "working_with/console/_index.md" >}}).
```

File: content/install_maintain/manager_architecture/components.md [hunk 4]
```diff
@@ -29,15 +29,15 @@ Rather than specifying the ports in each component's overview, ports are specifi
...
 
 * The network on which the CLI resides, which is potentially a user's `management network`.
 * The network on which the application resides, which is potentially a user's application network.
 
-Therefore, Cloudify requires only two entry points to its management environment:
+Therefore, {{< param product_name >}} requires only two entry points to its management environment:
 
 * Ports 80 / 443 for user rest-service/UI access via Nginx
-* Port 22 is exposed for SSH access, to enable remote access to the Cloudify management environment.
+* Port 22 is exposed for SSH access, to enable remote access to the C{{< param cfy_manager_name >}}.
```
Comment (qooban) [inline content/install_maintain/manager_architecture/components.md:40]:
```suggestion
* Port 22 is exposed for SSH access, to enable remote access to the {{< param cfy_manager_name >}}.
```

File: content/ops_guides/backup_restore_guide.md [hunk 5]
```diff
@@ -102,11 +102,11 @@ Snapshots of the Cloudify HA cluster should be taken at regular intervals (sugge
     "http://<manager-ip>/api/v3.1/snapshots/<snapshot-id>/restore"
     ```
 
-    Parameters specification available in the [Cloudify API documentation](http://docs.cloudify.co/api/latest/#restore-snapshot).
+    Parameters specification available in the [{{< param product_name >}} API documentation](http://docs.cloudify.co/api/latest/#restore-snapshot).
 
 1. Snapshot-restore status
 
-    **(Supported for Cloudify Manager 5.0.5 and above.)**
+    **(Supported for {{< param cfy_manager_name >}} 5.0.5 and above.)**
```
Comment (qooban) [inline content/ops_guides/backup_restore_guide.md:109]: In other places you removed references to specific versions. Maybe here it should be removed as well?

File: content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md [hunk 6]
```diff
@@ -1,113 +1,113 @@
 ---
 title: Cloudify cluster day 2 operations
-description: Guides for manging Cloudify HA clusters
+description: Cloudify HA clusters management guides
```
Comment (qooban) [inline content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md:3]:
```suggestion
description: HA clusters management guides
```

File: content/ops_guides/troubleshooting.md [hunk 7]
```diff
@@ -124,9 +124,9 @@ cfy plugins upload <plugin file path> -y plugin.yaml path
 # Troubleshooting techniques
 
 
-## How to run cloudify cli commands
+## How to run {{< param cfy_cli_name >}} commands
 
-In order to run the cloudify cli command 'cfy' you need to ensure that you have installed the Cloudify CLI
+In order to run the c{{< param cfy_cli_name >}} command 'cfy' you need to ensure that you have installed the {{< param cfy_cli_name >}}
```
Comment (qooban) [inline content/ops_guides/troubleshooting.md:129]:
```suggestion
In order to run the {{< param cfy_cli_name >}} command 'cfy' you need to ensure that you have installed the {{< param cfy_cli_name >}}
```

**Review by qooban — COMMENTED** [review]
Commented.

File: content/ops_guides/backup_restore_guide.md (hunk 5 above)
Comment (ofercloudify) [reply content/ops_guides/backup_restore_guide.md:109]: agree. fixed.

File: content/developer/writing_plugins/creating-your-own-plugin.md (hunk 1 above)
Comment (ofercloudify) [reply content/developer/writing_plugins/creating-your-own-plugin.md:619]: no, we cannot.

**Review by qooban — APPROVED** [review]

**Pull request merged**